GitCommands/Git/GitItemStatus.cs

```diff
@@ -53,6 +53,7 @@ private enum Flags
         public string RenameCopyPercentage { get; set; }
 
         public StagedStatus Staged { get; set; }
+        public bool RangeDiff { get; set; }
```

Comment:
Not sure, should this be another flag as those below, especially like `IsStatusOnly`?

Reply:
Yes it could be a flag. I choose not to as the other flags are git properties. IsStatusOnly is a special flag to show that this is not derived from Git (this could be separate too).

But I do not think there are many GitItemStatus at the same time, so a few bytes per file should not matter.
There are limited number of files in diff tab or similar, the RevTree only uses GitItemStatus when viewing files.

I do not mind changing this, we can add a comment separating properties from Git with the extra data

Reply:
I agree, this won't turn the scales.
(Though for merges of vendor branches, almost all files are in the diff. Then diff loading is really slow already without range-diff.)
Not requesting a change.

Reply:
Changing this, adding comments anyway
But this should probably be an item on the FileStatusItem instead. The problem with that is though that GitStatusItems are often isolated. The only special for RangeDiff would be for GetItemImageIndex() and that can be handled anyway, but there could be other usages.
(and that would require a separate property anyway, so no byte saved)